test/BoardFileTests.cpp

```diff
@@ -0,0 +1,35 @@
+#include <gtest/gtest.h>
+#include <string>
+#include "Utilities.cpp"
```

Comment:
A little bit late, but I just noticed it. 
@LordLukin utils functions should have their own header file or it can stay as it is?